R/readKane.R

```diff
@@ -0,0 +1,63 @@
+#' Raw Read for data corresponding to the Kane dataset
+#'
+#' Download and read in the raw data tables from Kane, E., Ping, C., Chapin, S. and 
+#' Valentine, D. with Bonanza LTER. doi:10.6073/pasta/d09433eee2cb6587eca672864cf7e90f
+#' 
+#' This is not post-processed or QA/QCed by this package.
+#'
+#' @param dataDir filename for download directory
+#' @param download boolean that will download the files from repository 
+#'
+#' @return a list with meta data and raw table reads. This will be 49 KB in size.
+#'
+#' @export
+
+dataDir <- '~/Desktop/Research/Kane' #edit to reflect personal directory
+
+
+#data read in for citation [2] not included
+  #url confirmation is needed, see ##TODO from 114_Kane.Rmd
+#KaneBiophysicalData is missing header information that could not be found online, contacting of researchers is necessary
+
+
+readKane2004 <- function(dataDir, download = TRUE, verbose = FALSE){
+ 
+  
+  url_table <- data.frame(File_Base = c('KaneSoilData.txt', 'KaneBiophysicalData.txt', 'KaneMetaData.xml'),
```

Comment:
Start variable names (including inside dataframes and lists) as lower case and then EITHER use upper case to denote a new word in the variable name (ie variableName) OR use an _ to separate the words. Syntactically correct but looks funny.